libs/feature-flags/src/lib/configcat.ts

```diff
@@ -15,18 +22,20 @@ export class Client implements FeatureFlagClient {
         'Trying to initialize configcat client without CONFIGCAT_SDK_KEY environment variable',
       )
     }
-    const ccConfig: IJSAutoPollOptions = {
+    const ccConfig: IAutoPollOptions = {
       dataGovernance: DataGovernance.EuOnly,
     }
     if (typeof window === 'undefined') {
-      this.configcat = eval('require')('configcat-node').createClient(
+      this.configcat = eval('require')('configcat-node').getClient(
         resolvedSdkKey,
+        null,
         ccConfig,
       )
     } else {
       // eslint-disable-next-line @typescript-eslint/no-var-requires
-      this.configcat = require('configcat-js').createClient(
+      this.configcat = require('configcat-js').getClient(
```

Comment:
Is the updated package, not an ES module? Just wondering if import() should be used instead

Reply:
This module is dynamically importing web vs nodejs version of configcat. We're trying not to run or bundle both.